app/controllers/users_controller.rb

```diff
@@ -50,6 +50,6 @@ def user_params
   end
 
   def phone_param
-    (params[:mobile_phone] || {})[:number]
+    (params[:mobile_phone] || {})[:full]
```

Comment:
The validation lib is set up to add a second hidden input that will contain the internationalised number, so 07856 253 263 will be turned (for the UK) into +447856253263 and stashed in this field on submit. This change allows us to capture the value from that field in the form, so that's what we store in the db.

The validation lib will reverse this and format the number nicely when we display it again.

Reply:
Thanks a lot for the explanation - hugely appreciated!  BTW in future might be helpful to actually stick it in a comment in the code since people are unlikely to come back to this PR.  So just to confirm I understand, we'll store the normalized version in the db not the user's version?

Reply:
Yep, the backend will be presented with the normalised version. It's only a weak validation of course, as the client could always modify the JS to do anything they want.

Reply:
I've added a comment, I should have done that in the first place.